task_router_activity_test.go

```diff
@@ -0,0 +1,46 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+	"testing"
+)
+
+func TestGetActivity(t *testing.T) {
+	t.Parallel()
+	client, server := getServer(taskRouterActivityResponse)
+	defer server.Close()
+
+	sid := "WAc74e6c39eb3080f8211d049a8b95611c"
+	name := "NewAvailableActivity"
+
+	activity, err := client.TaskRouter.TaskRouterActivity.Get(context.Background(), sid)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if activity.Sid != sid {
+		t.Errorf("activity: got sid %q, want %q", activity.Sid, sid)
+	}
+
+	if activity.FriendlyName != name {
+		t.Errorf("activity: got sid %q, want %q", activity.FriendlyName, name)
+	}
+}
+func TestCreateActivity(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping HTTP request in short mode")
+	}
+	t.Parallel()
+
+	data := url.Values{}
+	newname := "Some Activity"
+	data.Set("FriendlyName", newname)
+
+	acct, err := envClient.TaskRouter.TaskRouterActivity.Create(context.Background(), data)
```

Comment:
This name kinda "stutters" a little bit... we mention "TaskRouter" twice. 

How about `client.TaskRouter.Activities.Create`? We have "Calls" and "Messages" so "Activities" seems like a better fit than "Activity."

Reply:
Agreed, should we change `TaskRouterActivityService` as well? I think not as I quite like the verbosity of this name.

Reply:
For now I've just updated to `Activities` and not changed the service name, happy to do so if you see fit.

https://github.com/kevinburke/twilio-go/pull/29/commits/3abaf3a3a8d44f0092cbaac0362042d3e972ceef